rpm/python3-pytz.spec

```diff
@@ -0,0 +1,34 @@
+Summary:       Pytz
+Name:          pytz
+Version:       2022.1
+Release:       1
+License:       MIT
+Source0:       %{name}-%{version}.tar.gz
+URL:           https://pypi.org/project/pytz/
+Patch0:        0001-Use-system-zoneinfo.patch
+
+BuildRequires: python3-devel
+BuildRequires: python3-setuptools
+BuildRequires: rsync
+
+%description
+pytz brings the Olson tz database into Python
+
+%prep
+%autosetup -p1 -n %{name}-%{version}/%{name}
+
+%build
+make build
```

Comment:
please use `%make_build`

Reply:
If you insist, but the only way I can make that work is by defining _smp_mflags as -j1, which kind of defeats the purpose of %make_build, don't you think?

Reply:
I think it still makes sense. I wonder why calling make is needed at all when fedora just calls `%py3_build`.

Reply:
`%make_build -j1` should work

Reply:
> I think it still makes sense. I wonder why calling make is needed at all when fedora just calls `%py3_build`.

Fedora doesn't build python packages from git, they use sources from pypi. In this case, the sources in pypi is pretty much what you get after running make.

Reply:
Ah yes I forgot that part, thanks.